tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,29 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self):
+    return self._visualize_dataset(self._builder, 'train')
+
+  def _visualize_dataset(self, builder, split):
+    """Visualize dataset statistics with FACETS DIVE."""
```

Comment:
You should describe what is being returned (type, ...) and how to use it.
Maybe with an example?